### PR TITLE
Panorama fixes

### PR DESCRIPTION
**Pull request by fmatthew5876**
Fixes: #1534 
Fixes: #1550

I don't emulate the save game behavior of `RPG_RT`.

### CODE REVIEW

**Comment by fmatthew5876** [comment]
This fixes #1550 too

https://easyrpg.org/play/pr1551?game=yume_nikki&start-map-id=18&start-party=1&new-game

**Comment by Ghabry** [comment]
Jenkins: test this please

File: src/game_map.cpp [hunk 1]
```diff
@@ -1516,27 +1512,36 @@ void Game_Map::Parallax::Initialize(int width, int height) {
...
 	Params params = GetParallaxParams();
 
-	if (params.scroll_horz)
-		parallax_x = -map_info.position_x / 2;
-	else if (GetWidth() > 20 && parallax_width > SCREEN_TARGET_WIDTH)
-		parallax_x = -std::min(map_info.position_x,
-			(map_info.position_x / (SCREEN_TILE_WIDTH / TILE_SIZE)) * (parallax_width - SCREEN_TARGET_WIDTH) / (GetWidth() - 20));
-	else
-		parallax_x = 0;
+	// RPT_RT bug:
```
Comment (Ghabry) [inline src/game_map.cpp]: RTP_RT ;)

Comment (fmatthew5876) [reply src/game_map.cpp]: Fixed

**Pull request merged**